CONVERT_SERVICE/convert_service/convert_video/model/VideoConverterModel.py

```diff
@@ -0,0 +1,86 @@
+#
+# @videoConverter.py Copyright (c) 2021 Jalasoft.
```

Comment:
Please change the name of the file. Is not correct with the current file.

Reply:
Corrected! Thanks